shared/teams/channel/index.tsx

```diff
@@ -193,11 +193,9 @@ const Channel = (props: OwnProps) => {
     <Kb.Box style={styles.container}>
       {Styles.isMobile && <MobileHeader />}
       <Kb.SectionList
-        alwaysVounceVertical={false}
         renderSectionHeader={renderSectionHeader}
         stickySectionHeadersEnabled={Styles.isMobile}
         sections={sections}
-        style={styles.list}
```

Comment:
?

Reply:
this is fixed in https://github.com/keybase/client/pull/23049